cmd/private-org-sync/main.go

```diff
@@ -460,35 +460,73 @@ func main() {
 		git:                  gitExec,
 		failOnNonexistentDst: o.failOnNonexistentDst,
 	}
+	var errs []error
+	var locations []location
+
+	whitelist := o.WhitelistOptions.WhitelistConfig.Whitelist
+	if len(whitelist) > 0 {
```

Comment:
Can we make `getWhitelistedLocations` return an empty location list on an empty whitelist, so that the logic here can be simpler (without the `len(whitelist) > 0` condition)?